src/pages/newblog/NewBlog.jsx

```diff
@@ -0,0 +1,104 @@
+    */
+
+
+function NewBlog() {
+    const {register, handleSubmit, setValue, formState: {errors}} = useForm();
+    const [apiData, setApiData] = useState("0");
+    const [error, setError] = useState("");
+    const [loading, toggleLoading] = useState(false);
+
+
+    useEffect(() => {
+        register("created");
+        register("readTime");
+        register("comments");
+        register("shares");
```

Comment:
Hè? De register-functie van react-hook-form wordt gebruikt op invoer-elementen (waar ik ze ook zie staan, dus dat is goed) maar wat je hier doet is niet de bedoeling en heeft ook geen zin :)